docs/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/getporter/porter
+
+go 1.21.0
```

Comment:
netlify is mad and it wants this to match the format `1.21`- once it is passing on netlify though we will probably remove this and just update our go.mod :)

Reply:
Alright, also the `go.mod` file in root dir has a different go version `go 1.20`. Is there any chance that different versions of go is also causing the issue?

Reply:
I'm not sure, right now the error in netlify is:
`invalid go version '1.21.0': must match format 1.23`
So it's just upset that there is a minor version added. I think it would be safest to use the version in the main go.mod (`1.20`) for now :)

Reply:
Ok Thanks 👍🏼, I will do that.

Reply:
Thanks, Changing go version to 1.20 fixed the issue. 👍🏼